cheetah/particles/beam.py

```diff
@@ -8,6 +8,26 @@
     physical_constants["electron mass energy equivalent in MeV"][0] * 1e6
 )
 
+r"""
+Cheetah uses a 7D vector to describe the state of a particle.
+It contains the 6D phase space vector (x, px, y, yp, tau, p) and an additional
+dimension (always 1) for convenient calculations.
+
+The phase space vectors contain the canonical variables:
+- x: Position in x direction in meters.
+- px: Horizontal momentum normalized over the reference momentum (dimensionless).
+    $px = P_x / P_0$
+- y: Position in y direction in meters.
+- py: Vertical momentum normalized over the reference momentum (dimensionless).
+    $py = P_y / P_0$
+- tau: Position in longitudinal direction in meters, relative to the reference particle.
+    $\tau = ct - s/\beta_0$, where s is the position along the beamline.
+    In this notation, particle ahead of the reference particle will have negative $\tau$
+- p: Relative energy deviation from the reference particle (dimensionless).
+    $p = \frac{\Delta E}{p_0 C}$, where $p_0$ is the reference momentum.
+    $\Delta E = E - E_0$
+"""
```

Comment:
I think this should go in a different place.

Reply:
What's your suggestion? I initially put this one here before splitting the beam into different files with the intent to give people who read into the source code a better understanding.

Reply:
In that case I would make it part of the `Beam` docstring. This also makes more sense because you added the math formatting which would never render anywhere if it's not part of anything's docstring. I will commit a proposal and you let me know what you think.

Reply:
In the end this is mostly just a summary of the coordinate systems doc page placed so people going through the code stumble over it, right?

Reply:
Yes. It looks good to me now.